tests/unit/test_page_iterator.py

```diff
@@ -432,6 +434,67 @@ def test__get_next_page_bad_http_method(self):
         with pytest.raises(ValueError):
             iterator._get_next_page_response()
 
+    @pytest.mark.parametrize(
+        "page_size,max_results,pages",
+        [(3, None, False), (3, 8, False), (3, None, True), (3, 8, True)])
+    def test_page_size_items(self, page_size, max_results, pages):
+        path = "/foo"
+        NITEMS = 10
+        page_size = 3
```

Comment:
Looks like this is getting set twice, once via "parametrize" and once here.

Reply:
Fixed. Thanks!